docs/outputs.rst

```diff
@@ -150,8 +150,8 @@ in precise geographic correspondence, with varying mesh densities: 32k (default)
 **Grayordinates files**.
 CIFTI is a container format that holds both volumetric (regularly sampled in a grid)
 and surface (sampled on a triangular mesh) samples.
-Sub-cortical time series are volumetric (supported spaces: ``MNI152NLin6Asym``), while cortical
-time series are sampled to surface (supported spaces: ``fsLR``).
+Sub-cortical time series are volumetric (space: ``MNI152NLin6Asym``), while cortical
+time series are sampled to surface (space: ``fsLR``).
```

Comment:
```suggestion
time-series are sampled on surfaces projected from the [Glasser2016]_ template.
```

With the Glasser reference being:
> Glasser, M., Coalson, T., Robinson, E. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016). https://doi.org/10.1038/nature18933

Perhaps, it would be nice to also add a link to the cifti spec of HCP: https://www.nitrc.org/forum/attachment.php?attachid=333&group_id=454&forum_id=1955